modules/registry/examples_test.go

```diff
@@ -39,9 +38,10 @@ func ExampleRun() {
 }
 
 func ExampleRun_withAuthentication() {
+	ctx := context.Background()
 	// htpasswdFile {
```

Comment:
Let's include the creation of the ctx in the embedded docs:
```suggestion
	// htpasswdFile {
	ctx := context.Background()
```